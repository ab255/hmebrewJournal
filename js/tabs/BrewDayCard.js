import React, {Component} from 'react';
import {
  ScrollView,
  Text,
  View,
} from 'react-native';

class BrewDayCard extends Component {
  constructor(props) {
    super()
  }

  render() {
    return (
      <ScrollView>
        <View>
          <Text>Brew Date</Text>
          <Text>{this.props.brewDate}</Text>
        </View>
        <View>
          <Text>Start Time</Text>
          <Text>{this.props.timeStarted}</Text>
        </View>
        <View>
          <Text>Current weather</Text>
          <Text>{this.props.weather}</Text>
        </View>
        <View>
          <Text>Mash/Steep Notes</Text>
          <Text>{this.props.mashNotes}</Text>
        </View>
        <View>
          <Text>Boil Notes</Text>
          <Text>{this.props.boilNotes}</Text>
        </View>
        <View>
          <Text>Post Boil Notes</Text>
          <Text>{this.props.postBoilNotes}</Text>
        </View>
        <View>
          <Text>Original Gravity</Text>
          <Text>{this.props.ogReading}</Text>
        </View>
      </ScrollView>
    )
  }
}

export default BrewDayCard;
