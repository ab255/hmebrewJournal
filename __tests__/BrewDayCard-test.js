import 'react-native';
import React from 'react';
import BrewDayCard from '../js/tabs/BrewDayCard.js';

import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <BrewDayCard />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
