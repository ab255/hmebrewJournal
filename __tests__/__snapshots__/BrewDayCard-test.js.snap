exports[`test renders correctly 1`] = `
<ScrollView
  style={
    Object {
      "padding": 10,
    }
  }>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      Brew Date
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      }>
      November 10th 2016
    </Text>
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      Start Time
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      } />
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      Current weather
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      }>
       ÂºF
    </Text>
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      Mash/Steep Notes
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      } />
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      Boil Notes
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      } />
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      Post Boil Notes
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      } />
  </View>
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#911F27",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 7,
          "textDecorationLine": "underline",
        }
      }>
      Original Gravity
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#630A10",
          "fontSize": 16,
          "marginBottom": 10,
        }
      } />
  </View>
</ScrollView>
`;
